scripts/eval_trajectory_to_lisdf.py

```diff
@@ -56,16 +59,76 @@ def _main() -> None:
     joint_names = robot.arm_joint_names
     assert len(joint_names) == joint_arr.shape[1]
 
-    # For now, we just convert the trajectory into one big JointSpacePath. In
-    # the future, we may want to chunk it up.
-    duration = args.time_per_conf * len(joint_arr)
-    command = JointSpacePath.from_waypoints_np_array(
-        joint_arr,
-        joint_names,
-        duration=duration,
-        label=args.input,
-    )
-    lisdf_plan = LISDFPlan(commands=[command], lisdf_problem="no_problem")
+    # Convert the joint array into a list of (arm joint state, gripper state)
+    # where the gripper state is GripperPosition.open or GripperPosition.close
+    # and the arm joint state is the original joints with grippers removed.
+    gripper_idxs = [robot.left_finger_joint_idx, robot.right_finger_joint_idx]
+    arm_joint_names = list(np.delete(joint_names, gripper_idxs))
```

Comment:
Just a general comment that doesn't need addressing.

The naming here isn't super satisfying given we have `robot.arm_joint_names` as well. We might want to think about renaming the latter so there are arm joints (for the actual arm) and gripper joints for the gripper, and they some name that encapsulates both.